From 9aabb79130a8ccfd56ef34f345c0462dc45c45e5 Mon Sep 17 00:00:00 2001
From: hololeap <hololeap@users.noreply.github.com>
Date: Sun, 23 Apr 2023 14:26:46 -0600
Subject: [PATCH 1/1] Add executable flag

Flag allows one to toggle whether or not to build the 'floskell'
executable.

Signed-off-by: hololeap <hololeap@users.noreply.github.com>
---
 floskell.cabal | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/floskell.cabal b/floskell.cabal
index f0fc110..e2ace8e 100644
--- a/floskell.cabal
+++ b/floskell.cabal
@@ -72,7 +72,13 @@ library
         unordered-containers >=0.2.8.0 && <0.3,
         utf8-string >=1.0.1.1 && <1.1
 
+flag executable
+    description: Build the floskell executable
+    default: False
+
 executable floskell
+    if !flag(executable)
+        buildable: False
     main-is:          Main.hs
     hs-source-dirs:   src/main
     default-language: Haskell2010
-- 
2.39.2

